fix misspells and minor cleanup (#44)
squid-cache/squid-cache.github.io: docs/SquidFaq/SquidProfiling.md

Squid is a CPU-intensive application (since, after all, it spends all of
its time processing incoming data and generating data to send.) But

> there are many different types of CPU usage which can identify what you're

Reviewer: another contraction to remove
```suggestion
there are many different types of CPU usage which can identify what you are
```

The best thing you can do to identify where all your CPU usage is going
is to use a process or system profiler. Personally, I use oprofile.

> gprof isn't at all accurate with modern CPU clockspeeds. There are other

Reviewer: another contraction to remove
```suggestion
gprof is not at all accurate with modern CPU clockspeeds. There are other
```

options - hwpmc under FreeBSD, for example, can do somewhat what
oprofile can but it currently has trouble getting any samples from Squid

> in userspace. Grr. *perfmon* is also an option if you don't have root

Reviewer: more contractions
```suggestion
in userspace. Grr. *perfmon* is also an option if you do not have root
```

> Here's how I use oprofile:

Reviewer: and more contractions
```suggestion
Here is how I use oprofile:
```

Just remember:

> - Make sure you've got the debugging libraries and library symbols

Reviewer: and more contraction
```suggestion
- Make sure you have got the debugging libraries and library symbols
```

>     installed - under Ubuntu that's 'libc6-dbg'.

Reviewer 2: ```suggestion
    installed - under Ubuntu that is 'libc6-dbg'.
```

> - Don't try using it under timer interrupt mode, it'll suffer similar

Reviewer: and more contractions
```suggestion
- Do not try using it under timer interrupt mode, it will suffer similar
```
